### PR TITLE
Make nerve service namespace available

### PR DESCRIPTION
**Pull request by dichiarafrancesco**
This feature is capital when someone wants to enable the services whitelist in NerveHTTPD collector and filter by service name and namespace!

### CODE REVIEW

File: src/fullerite/util/nerve_config.go [hunk 1]
```diff
@@ -29,11 +29,16 @@ type nerveConfigData struct {
 	Services map[string]map[string]interface{}
 }
 
+type NerveService struct {
```
Comment (gnufied) [inline src/fullerite/util/nerve_config.go]: missing constant doc.

File: src/fullerite/collector/nerve_httpd_test.go [hunk 2]
```diff
@@ -153,5 +152,74 @@ func TestNerveHTTPDCollect(t *testing.T) {
...
+	flag := true
+	for flag == true {
+		select {
+		case metric := <-inst.Channel():
+			actual = append(actual, metric)
+		case <-time.After(2 * time.Second):
+			flag = false
+			break
+		}
+	}
```
Comment (gnufied) [inline src/fullerite/collector/nerve_httpd_test.go]: Do you think this test is vulnerable to timing problems? We are essentially saying we should be able to read all metrics within 2 seconds.

**Comment by gnufied** [comment]
Apart from above 2 very minor problems, it looks good. fix & ship!

File: src/fullerite/collector/nerve_httpd_test.go (hunk 2 above)
Comment (dichiarafrancesco) [inline src/fullerite/collector/nerve_httpd_test.go]: We are saying we should be able to read EACH metric within 2 seconds. So i think we are not vulnerable ;)

**Comment by gnufied** [comment]
:shipit:

**Pull request merged**